diff -burN FlightGear-0.9.6.orig/src/Airports/Makefile.am FlightGear-0.9.6/src/Airports/Makefile.am
--- FlightGear-0.9.6.orig/src/Airports/Makefile.am	2004-11-01 17:13:02.183256960 +0100
+++ FlightGear-0.9.6/src/Airports/Makefile.am	2004-11-01 17:13:54.304333352 +0100
@@ -7,6 +7,7 @@
 	simple.cxx simple.hxx
 
 calc_loc_SOURCES = calc_loc.cxx
-calc_loc_LDADD = -lsgmath -lsgdebug -lsgmisc -lz $(base_LIBS)
+calc_loc_LDADD = -lsgmath -lsgdebug -lsgmisc -lz $(base_LIBS) \
+		 -lsgprops -lsgstructure -lsgxml
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -burN FlightGear-0.9.6.orig/src/FDM/YASim/Makefile.am FlightGear-0.9.6/src/FDM/YASim/Makefile.am
--- FlightGear-0.9.6.orig/src/FDM/YASim/Makefile.am	2004-11-01 17:13:02.016282344 +0100
+++ FlightGear-0.9.6/src/FDM/YASim/Makefile.am	2004-11-01 17:14:28.542128416 +0100
@@ -43,9 +43,9 @@
 
 yasim_SOURCES = yasim-test.cpp $(SHARED_SOURCE_FILES)
 
-yasim_LDADD = -lsgxml -lsgprops -lsgdebug -lsgmisc -lsgstructure -lplibssg -lplibul -lplibsg $(opengl_LIBS) $(base_LIBS)
+yasim_LDADD = -lsgxml -lsgprops -lsgdebug -lsgmisc -lsgstructure -lplibssg -lplibul -lplibsg $(opengl_LIBS) $(base_LIBS) -lz
 
 proptest_SOURCES = proptest.cpp $(SHARED_SOURCE_FILES)
-proptest_LDADD = -lsgxml -lsgprops -lsgdebug -lsgmisc -lsgstructure -lplibssg -lplibul -lplibsg $(opengl_LIBS) $(base_LIBS)
+proptest_LDADD = -lsgxml -lsgprops -lsgdebug -lsgmisc -lsgstructure -lplibssg -lplibul -lplibsg $(opengl_LIBS) $(base_LIBS) -lz
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -burN FlightGear-0.9.6.orig/src/Main/Makefile.am FlightGear-0.9.6/src/Main/Makefile.am
--- FlightGear-0.9.6.orig/src/Main/Makefile.am	2004-11-01 17:13:02.143263040 +0100
+++ FlightGear-0.9.6/src/Main/Makefile.am	2004-11-01 17:15:10.865694256 +0100
@@ -104,6 +104,6 @@
 metar_LDADD = \
         -lsgenvironment -lsgio -lsgbucket -lsgmisc -lsgstructure -lsgdebug \
         -lplibnet -lplibul $(network_LIBS) \
-        -lz $(base_LIBS)
+        -lz $(base_LIBS) -lsgserial -lsgprops -lsgxml
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -burN FlightGear-0.9.6.orig/src/Navaids/Makefile.am FlightGear-0.9.6/src/Navaids/Makefile.am
--- FlightGear-0.9.6.orig/src/Navaids/Makefile.am	2004-11-01 17:13:02.239248448 +0100
+++ FlightGear-0.9.6/src/Navaids/Makefile.am	2004-11-01 17:15:37.842593144 +0100
@@ -15,6 +15,6 @@
 # testnavs_LDADD = \
 # 	libNavaids.a \
 # 	-lsgtiming -lsgmath -lsgmisc -lsgdebug -lsgmagvar -lsgxml \
-# 	-lsgstructure $(base_LIBS) -lz
+# 	-lsgstructure $(base_LIBS) -lz -lsgprops
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -burN FlightGear-0.9.6.orig/tests/Makefile.am FlightGear-0.9.6/tests/Makefile.am
--- FlightGear-0.9.6.orig/tests/Makefile.am	2004-11-01 17:13:02.274243128 +0100
+++ FlightGear-0.9.6/tests/Makefile.am	2004-11-01 17:15:59.265336392 +0100
@@ -17,4 +17,4 @@
 test_text_SOURCES = test-text.cxx
 
 test_up_SOURCES = test-up.cxx
-test_up_LDADD = -lsgmath -lsgxml -lsgmisc -lsgdebug -lsgstructure -lsgtiming -lplibsg -lplibul -lz $(base_LIBS)
+test_up_LDADD = -lsgmath -lsgxml -lsgmisc -lsgdebug -lsgstructure -lsgtiming -lplibsg -lplibul -lz $(base_LIBS) -lsgprops
--- FlightGear-0.9.9.org/src/GUI/Makefile.am	2005-07-30 16:20:24.000000000 +0200
+++ FlightGear-0.9.9/src/GUI/Makefile.am	2005-12-07 22:09:58.000000000 +0100
@@ -22,4 +22,6 @@
 
 layout_test_LDADD = libGUI.a \
                     -lsgprops -lsgdebug -lsgstructure -lsgmisc -lsgxml \
-                    -lplibpw -lplibpu -lplibfnt -lplibul $(opengl_LIBS)
+                    -lplibpw -lplibpu -lplibfnt -lplibul $(opengl_LIBS) \
+		    -lxml
+
--- FlightGear-0.9.10/utils/GPSsmooth/Makefile.am.orig	2006-08-16 20:29:39.000000000 -0700
+++ FlightGear-0.9.10/utils/GPSsmooth/Makefile.am	2006-08-16 20:36:00.000000000 -0700
@@ -5,7 +5,7 @@
 	gps_main.cxx
 
 GPSsmooth_LDADD = \
-	-lsgtiming -lsgmisc -lsgdebug -lplibnet -lplibul \
+	-lsgtiming -lsgmisc -lsgdebug -lplibnet -lplibul -lsgprops -lsgxml -lsgmisc -lsgstructure \
 	$(joystick_LIBS) $(network_LIBS) $(base_LIBS) -lz
 
 MIDGsmooth_SOURCES = \
@@ -13,7 +13,7 @@
 	MIDG_main.cxx
 
 MIDGsmooth_LDADD = \
-	-lsgio -lsgserial -lsgtiming -lsgmath -lsgbucket -lsgmisc -lsgdebug \
+	-lsgio -lsgserial -lsgtiming -lsgmath -lsgbucket -lsgmisc -lsgdebug -lsgstructure -lsgprops -lsgxml \
 	-lplibnet -lplibul \
 	$(joystick_LIBS) $(network_LIBS) $(base_LIBS) -lz
 
--- FlightGear-0.9.10/utils/Modeller/Makefile.am.orig	2006-08-16 20:36:44.000000000 -0700
+++ FlightGear-0.9.10/utils/Modeller/Makefile.am	2006-08-16 20:38:58.000000000 -0700
@@ -8,4 +8,6 @@
 animassist_LDADD = $(base_LIBS)
 
 normalmap_SOURCES = normalmap.cxx
-normalmap_LDADD = -lsgscreen $(opengl_LIBS) $(base_LIBS) -lz
+normalmap_LDADD = -lsgscreen -lsgdebug \
+	-lplibssg -lplibsg -lplibul \
+	$(opengl_LIBS) $(base_LIBS) -lz
--- FlightGear-0.9.10/src/Main/metar_main.cxx.orig	2006-08-16 20:26:14.000000000 -0700
+++ FlightGear-0.9.10/src/Main/metar_main.cxx	2006-08-16 20:26:55.000000000 -0700
@@ -28,6 +28,10 @@
 #include <simgear/environment/metar.hxx>
 #include <simgear/structure/exception.hxx>
 
+// Not really used there definitions - just to fix missing symbol in libsgsky.so.
+class SGSky;
+SGSky *thesky;
+
 using namespace std;
 
 // text color
--- FlightGear-0.9.10/src/Main/Makefile.am.mkorig	2006-08-17 14:09:38.000000000 -0700
+++ FlightGear-0.9.10/src/Main/Makefile.am	2006-08-17 14:16:55.000000000 -0700
@@ -106,8 +106,9 @@
 metar_SOURCES = metar_main.cxx
 
 metar_LDADD = \
-        -lsgenvironment -lsgio -lsgbucket -lsgmisc -lsgstructure -lsgdebug \
-        -lplibnet -lplibul $(network_LIBS) \
-        -lz $(base_LIBS) -lsgserial -lsgprops -lsgxml
+        -lsgenvironment -lsgio -lsgbucket -lsgmisc -lsgstructure -lsgdebug -lsgsky  \
+        -lplibnet -lplibul -lplibsg -lplibssg -lopenal $(network_LIBS) \
+        -lz -lGLU $(base_LIBS) -lsgserial -lsgprops -lsgxml -lsgscreen -lsgsound -lsgmath
+
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src -I$(top_srcdir)/src/FDM/JSBSim
