diff -urN FlightGear-0.7.10.orig/src/ATC/Makefile.am FlightGear-0.7.10/src/ATC/Makefile.am
--- FlightGear-0.7.10.orig/src/ATC/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/ATC/Makefile.am	Fri May 10 23:11:50 2002
@@ -12,8 +12,5 @@
 	AIEntity.hxx AIEntity.cxx \
 	AILocalTraffic.hxx AILocalTraffic.cxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Aircraft/Makefile.am FlightGear-0.7.10/src/Aircraft/Makefile.am
--- FlightGear-0.7.10.orig/src/Aircraft/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Aircraft/Makefile.am	Fri May 10 23:12:01 2002
@@ -2,8 +2,5 @@
 
 libAircraft_a_SOURCES = aircraft.cxx aircraft.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Airports/Makefile.am FlightGear-0.7.10/src/Airports/Makefile.am
--- FlightGear-0.7.10.orig/src/Airports/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Airports/Makefile.am	Fri May 10 23:12:29 2002
@@ -12,8 +12,5 @@
 genrunways_SOURCES = genrunways.cxx
 genrunways_LDADD = libAirports.a -lsgdebug -lsgmisc -lsgxml -lmk4 -lz
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Autopilot/Makefile.am FlightGear-0.7.10/src/Autopilot/Makefile.am
--- FlightGear-0.7.10.orig/src/Autopilot/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Autopilot/Makefile.am	Fri May 10 23:12:45 2002
@@ -4,8 +4,5 @@
 	auto_gui.cxx auto_gui.hxx \
 	newauto.cxx newauto.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Cockpit/Makefile.am FlightGear-0.7.10/src/Cockpit/Makefile.am
--- FlightGear-0.7.10.orig/src/Cockpit/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Cockpit/Makefile.am	Fri May 10 23:12:56 2002
@@ -12,10 +12,6 @@
 	radiostack.cxx radiostack.hxx \
 	steam.cxx steam.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
 
 SUBDIRS = built_in
diff -urN FlightGear-0.7.10.orig/src/Cockpit/built_in/Makefile.am FlightGear-0.7.10/src/Cockpit/built_in/Makefile.am
--- FlightGear-0.7.10.orig/src/Cockpit/built_in/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Cockpit/built_in/Makefile.am	Fri May 10 23:13:10 2002
@@ -3,9 +3,5 @@
 libBuilt_in_a_SOURCES = \
 	FGMagRibbon.cxx FGMagRibbon.hxx 
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
 
diff -urN FlightGear-0.7.10.orig/src/Controls/Makefile.am FlightGear-0.7.10/src/Controls/Makefile.am
--- FlightGear-0.7.10.orig/src/Controls/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Controls/Makefile.am	Fri May 10 23:13:21 2002
@@ -2,8 +2,5 @@
 
 libControls_a_SOURCES = controls.cxx controls.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Environment/Makefile.am FlightGear-0.7.10/src/Environment/Makefile.am
--- FlightGear-0.7.10.orig/src/Environment/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Environment/Makefile.am	Fri May 10 23:13:37 2002
@@ -7,8 +7,5 @@
 libEnvironment_a_SOURCES = environment.cxx environment.hxx \
 	environment_mgr.cxx environment_mgr.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/FDM/Balloon/Makefile.am FlightGear-0.7.10/src/FDM/Balloon/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/Balloon/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/Balloon/Makefile.am	Fri May 10 23:13:49 2002
@@ -2,8 +2,5 @@
 
 libBalloon_a_SOURCES = BalloonSim.cpp BalloonSim.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/FDM/JSBSim/Makefile.am FlightGear-0.7.10/src/FDM/JSBSim/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/JSBSim/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/JSBSim/Makefile.am	Fri May 10 23:14:05 2002
@@ -54,11 +54,6 @@
 
 # noinst_PROGRAMS = testJSBsim
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-DEFS += -DFGFS
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
 AM_CXXFLAGS = -DFGFS
-endif
 
diff -urN FlightGear-0.7.10.orig/src/FDM/JSBSim/filtersjb/Makefile.am FlightGear-0.7.10/src/FDM/JSBSim/filtersjb/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/JSBSim/filtersjb/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/JSBSim/filtersjb/Makefile.am	Fri May 10 23:14:38 2002
@@ -12,11 +12,6 @@
 	FGSummer.cpp FGSummer.h \
 	FGSwitch.cpp FGSwitch.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-DEFS += -DFGFS
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
 AM_CXXFLAGS = -DFGFS
-endif
 
diff -urN FlightGear-0.7.10.orig/src/FDM/LaRCsim/Makefile.am FlightGear-0.7.10/src/FDM/LaRCsim/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/LaRCsim/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/LaRCsim/Makefile.am	Fri May 10 23:15:01 2002
@@ -32,8 +32,5 @@
 	$(AIRCRAFT_MODEL) \
 	ls_interface.c ls_interface.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/FDM/Makefile.am FlightGear-0.7.10/src/FDM/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/Makefile.am	Fri May 10 23:15:16 2002
@@ -14,8 +14,5 @@
 	MagicCarpet.cxx MagicCarpet.hxx \
 	NullFDM.cxx NullFDM.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/FDM/UIUCModel/Makefile.am FlightGear-0.7.10/src/FDM/UIUCModel/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/UIUCModel/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/UIUCModel/Makefile.am	Fri May 10 23:15:28 2002
@@ -52,8 +52,5 @@
                         uiuc_warnings_errors.cpp uiuc_warnings_errors.h \
                         uiuc_wrapper.cpp uiuc_wrapper.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/FDM/YASim/Makefile.am FlightGear-0.7.10/src/FDM/YASim/Makefile.am
--- FlightGear-0.7.10.orig/src/FDM/YASim/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/FDM/YASim/Makefile.am	Fri May 10 23:15:39 2002
@@ -23,8 +23,5 @@
         Vector.hpp \
         Wing.cpp Wing.hpp
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/GUI/Makefile.am FlightGear-0.7.10/src/GUI/Makefile.am
--- FlightGear-0.7.10.orig/src/GUI/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/GUI/Makefile.am	Fri May 10 23:15:51 2002
@@ -16,8 +16,5 @@
 	sgVec3Slider.cxx sgVec3Slider.hxx \
 	trackball.c trackball.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Input/Makefile.am FlightGear-0.7.10/src/Input/Makefile.am
--- FlightGear-0.7.10.orig/src/Input/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Input/Makefile.am	Fri May 10 23:16:04 2002
@@ -12,8 +12,5 @@
 
 fgjs_LDADD = $(audio_LIBS)
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Main/Makefile.am FlightGear-0.7.10/src/Main/Makefile.am
--- FlightGear-0.7.10.orig/src/Main/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Main/Makefile.am	Fri May 10 23:16:20 2002
@@ -90,10 +90,6 @@
 
 fgfs_CFLAGS = -DTHIS_IS_A_TEST
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
 
 
diff -urN FlightGear-0.7.10.orig/src/Model/Makefile.am FlightGear-0.7.10/src/Model/Makefile.am
--- FlightGear-0.7.10.orig/src/Model/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Model/Makefile.am	Fri May 10 23:16:29 2002
@@ -4,8 +4,5 @@
                      modelmgr.cxx modelmgr.hxx \
                      acmodel.cxx acmodel.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Navaids/Makefile.am FlightGear-0.7.10/src/Navaids/Makefile.am
--- FlightGear-0.7.10.orig/src/Navaids/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Navaids/Makefile.am	Fri May 10 23:16:40 2002
@@ -14,8 +14,5 @@
 	-lsgtiming -lsgmath -lsgmisc -lsgdebug -lsgmagvar -lsgxml \
 	-lz
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Network/Makefile.am FlightGear-0.7.10/src/Network/Makefile.am
--- FlightGear-0.7.10.orig/src/Network/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Network/Makefile.am	Fri May 10 23:16:51 2002
@@ -26,8 +26,5 @@
 	ray.cxx ray.hxx \
 	rul.cxx rul.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/NetworkOLK/Makefile.am FlightGear-0.7.10/src/NetworkOLK/Makefile.am
--- FlightGear-0.7.10.orig/src/NetworkOLK/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/NetworkOLK/Makefile.am	Fri May 10 23:17:02 2002
@@ -3,8 +3,5 @@
 libNetworkOLK_a_SOURCES = \
 	net_send.cxx net_hud.cxx network.cxx network.h fgd.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Objects/Makefile.am FlightGear-0.7.10/src/Objects/Makefile.am
--- FlightGear-0.7.10.orig/src/Objects/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Objects/Makefile.am	Fri May 10 23:17:11 2002
@@ -8,8 +8,5 @@
 	obj.cxx obj.hxx \
 	texload.c texload.h colours.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Scenery/Makefile.am FlightGear-0.7.10/src/Scenery/Makefile.am
--- FlightGear-0.7.10.orig/src/Scenery/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Scenery/Makefile.am	Fri May 10 23:17:21 2002
@@ -8,8 +8,5 @@
 	tileentry.cxx tileentry.hxx \
 	tilemgr.cxx tilemgr.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Sound/Makefile.am FlightGear-0.7.10/src/Sound/Makefile.am
--- FlightGear-0.7.10.orig/src/Sound/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Sound/Makefile.am	Fri May 10 23:17:32 2002
@@ -7,8 +7,5 @@
 	morse.cxx morse.hxx \
 	soundmgr.cxx soundmgr.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/Time/Makefile.am FlightGear-0.7.10/src/Time/Makefile.am
--- FlightGear-0.7.10.orig/src/Time/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/Time/Makefile.am	Fri May 10 23:17:42 2002
@@ -8,8 +8,5 @@
 	sunpos.cxx sunpos.hxx \
 	tmp.cxx tmp.hxx
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
diff -urN FlightGear-0.7.10.orig/src/WeatherCM/Makefile.am FlightGear-0.7.10/src/WeatherCM/Makefile.am
--- FlightGear-0.7.10.orig/src/WeatherCM/Makefile.am	Fri May 10 23:08:45 2002
+++ FlightGear-0.7.10/src/WeatherCM/Makefile.am	Fri May 10 23:17:54 2002
@@ -19,8 +19,5 @@
         linintp2.cpp linintp2.h \
         sphrintp.cpp sphrintp.h
 
-if OLD_AUTOMAKE
-INCLUDES += -I$(top_srcdir) -I$(top_srcdir)/src
-else
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
-endif
+
--- FlightGear-0.7.10.origtests/Makefile.am~	Tue Mar  5 01:20:50 2002
+++ FlightGear-0.7.10/tests/Makefile.am	Fri May 10 23:31:04 2002
@@ -21,4 +21,4 @@
 test_text_LDADD = $(base_LIBS)
 
 test_up_SOURCES = test-up.cxx
-test_up_LDADD = -lsgmath -lsgdebug -lplibsg $(base_LIBS)
+test_up_LDADD = -lsgmath -lsgdebug -lplibsg -lsgmisc -lz -lsgxml $(base_LIBS)
