diff -urN aaa/FlightGear-0.9.2/src/FDM/YASim/Makefile.am FlightGear-0.9.2/src/FDM/YASim/Makefile.am
--- aaa/FlightGear-0.9.2/src/FDM/YASim/Makefile.am	2003-05-20 20:28:54.000000000 +0200
+++ FlightGear-0.9.2/src/FDM/YASim/Makefile.am	2003-08-02 20:40:37.000000000 +0200
@@ -55,6 +55,6 @@
         Vector.hpp \
         Wing.cpp Wing.hpp
 
-yasim_LDADD = -lsgxml -lsgprops -lsgmisc -lsgdebug
+yasim_LDADD = -lsgxml -lsgprops -lsgmisc -lsgdebug -lz
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -urN aaa/FlightGear-0.9.2/tests/Makefile.am FlightGear-0.9.2/tests/Makefile.am
--- aaa/FlightGear-0.9.2/tests/Makefile.am	2003-05-20 20:30:06.000000000 +0200
+++ FlightGear-0.9.2/tests/Makefile.am	2003-08-02 20:39:55.000000000 +0200
@@ -21,4 +21,4 @@
 test_text_LDADD = $(base_LIBS)
 
 test_up_SOURCES = test-up.cxx
-test_up_LDADD = -lsgmath -lsgdebug -lplibsg -lplibul $(base_LIBS)
+test_up_LDADD = -lsgmath -lsgdebug -lplibsg -lplibul -lsgmisc -lsgxml -lz $(base_LIBS)
