--- flightgear-2.10.0/data/Shaders/urban.frag.old	2013-01-28 17:23:45.000000000 +0100
+++ flightgear-2.10.0/data/Shaders/urban.frag	2013-03-27 10:12:06.993411375 +0100
@@ -90,8 +90,10 @@
                 #ifdef USE_QDM_ASCEND_INTERVAL
                 if(frac(level*0.5) > EPSILON)
                   level++;
-                #elseif USE_QDM_ASCEND_CONST
-                 level++;
+                #else
+                  #ifdef USE_QDM_ASCEND_CONST
+                    level++;
+                  #endif
                 #endif
             }
             p2 = p + v * d;
