diff -urN aaa/FlightGear-0.9.3/src/FDM/YASim/Makefile.am FlightGear-0.9.3/src/FDM/YASim/Makefile.am
--- aaa/FlightGear-0.9.3/src/FDM/YASim/Makefile.am	2003-10-16 16:40:26.000000000 +0200
+++ FlightGear-0.9.3/src/FDM/YASim/Makefile.am	2003-11-11 01:57:06.000000000 +0100
@@ -39,6 +39,6 @@
 
 yasim_SOURCES = yasim-test.cpp $(SHARED_SOURCES)
 
-yasim_LDADD = -lsgxml -lsgprops -lsgdebug -lsgstructure  $(base_LIBS)
+yasim_LDADD = -lsgxml -lsgprops -lsgdebug -lsgstructure -lsgmisc -lsgtiming  $(base_LIBS) -lz
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -urN aaa/FlightGear-0.9.3/src/Navaids/Makefile.am FlightGear-0.9.3/src/Navaids/Makefile.am
--- aaa/FlightGear-0.9.3/src/Navaids/Makefile.am	2003-08-29 19:35:49.000000000 +0200
+++ FlightGear-0.9.3/src/Navaids/Makefile.am	2003-11-11 02:02:33.000000000 +0100
@@ -11,7 +11,7 @@
 testnavs_SOURCES = testnavs.cxx
 testnavs_LDADD = \
 	libNavaids.a \
-	-lsgtiming -lsgmath -lsgmisc -lsgdebug -lsgmagvar -lsgxml \
+	-lsgtiming -lsgmath -lsgmisc -lsgdebug -lsgmagvar -lsgxml -lsgstructure \
 	$(base_LIBS) -lz
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/src
diff -urN aaa/FlightGear-0.9.3/tests/Makefile.am FlightGear-0.9.3/tests/Makefile.am
--- aaa/FlightGear-0.9.3/tests/Makefile.am	2003-08-29 19:35:51.000000000 +0200
+++ FlightGear-0.9.3/tests/Makefile.am	2003-11-11 01:57:06.000000000 +0100
@@ -17,4 +17,4 @@
 test_text_SOURCES = test-text.cxx
 
 test_up_SOURCES = test-up.cxx
-test_up_LDADD = -lsgmath -lsgdebug -lplibsg -lplibul $(base_LIBS)
+test_up_LDADD = -lsgmath -lsgdebug -lplibsg -lplibul -lsgmisc -lsgxml -lsgstructure -lsgtiming -lz $(base_LIBS)
