--- flightgear-3.2.0/src/Model/panelnode.cxx.orig	2014-08-17 15:40:24.000000000 +0200
+++ flightgear-3.2.0/src/Model/panelnode.cxx	2014-11-02 12:27:29.031193511 +0100
@@ -10,6 +10,7 @@
 #include <osg/Geode>
 #include <osg/Switch>
 #include <osg/BlendFunc>
+#include <osg/Version>
 
 #include <simgear/compiler.h>
 #include <simgear/structure/exception.hxx>
@@ -315,7 +316,11 @@
 }
 
 osg::BoundingBox
+#if OSG_VERSION_LESS_THAN(3,3,2)
 FGPanelNode::computeBound() const
+#else
+FGPanelNode::computeBoundingBox() const
+#endif
 {
 
   osg::Vec3 coords[3];
--- flightgear-3.2.0/src/Model/panelnode.hxx.orig	2014-08-17 15:40:24.000000000 +0200
+++ flightgear-3.2.0/src/Model/panelnode.hxx	2014-11-02 12:27:11.551194127 +0100
@@ -4,6 +4,7 @@
 #include <osg/Vec3>
 #include <osg/Matrix>
 #include <osg/Drawable>
+#include <osg/Version>
 
 #include <memory>
 #include <simgear/structure/SGSharedPtr.hxx>
@@ -30,7 +31,11 @@
     { drawImplementation(*renderInfo.getState()); }
   
     void drawImplementation(osg::State& state) const;
+#if OSG_VERSION_LESS_THAN(3,3,2)
     virtual osg::BoundingBox computeBound() const;
+#else
+    virtual osg::BoundingBox computeBoundingBox() const;
+#endif
 
     /** Return true, FGPanelNode does support accept(PrimitiveFunctor&). */
     virtual bool supports(const osg::PrimitiveFunctor&) const { return true; }
